examples/bri-3/src/bri/zeroKnowledgeProof/services/circuit/snarkjs/snarkjs.service.spec.ts

```diff
@@ -0,0 +1,160 @@
+import { SnarkjsCircuitService } from './snarkjs.service';
+import { Transaction } from '../../../../transactions/models/transaction';
+import { TransactionStatus } from '../../../../transactions/models/transactionStatus.enum';
+import { BpiSubjectAccount } from '../../../../identity/bpiSubjectAccounts/models/bpiSubjectAccount';
+import { BpiSubject } from '../../../../identity/bpiSubjects/models/bpiSubject';
+import {
+  BpiSubjectRole,
+  BpiSubjectRoleName,
+} from '../../../../identity/bpiSubjects/models/bpiSubjectRole';
+import { Witness } from '../../../models/witness';
+import { ed25519 } from '@noble/curves/ed25519';
+import * as circomlib from 'circomlibjs';
+import * as crypto from 'crypto';
+import 'dotenv/config';
+
+jest.setTimeout(20000);
+describe('SnarkjsService', () => {
+  const snarkjs = new SnarkjsCircuitService();
+  let inputs: any;
+  let witness: Witness;
+
+  //REMOVE THIS TEST: Empty test
```

Comment:
seems like there are some todos and commented out code in this file, could you please update it to be ready to check?